azure-pipelines/azure-pipelines.yml

```diff
@@ -50,87 +50,87 @@ stages:
         inputs:
           command: 'get'
           arguments: '-d'
-          workingDirectory: '$(System.DefaultWorkingDirectory)/deviceshifu/pkg/deviceshifu'
+          workingDirectory: '$(System.DefaultWorkingDirectory)/pkg/deviceshifu/deviceshifuHTTP'
```

Comment:
Should execute this in the default dir `$(System.DefaultWorkingDirectory)`

Reply:
`go get -d `need to execute on certain pkg, so it cannot be executed on `$(System.DefaultWorkingDirectory)`